Dockerfile.prow

```diff
@@ -6,7 +6,7 @@ WORKDIR /workspace
 COPY . .
 
 RUN go mod vendor
-RUN go install github.com/prometheus/promu@v0.17.0 && make common-build
+RUN go install github.com/prometheus/promu@v0.17.0 && promu build -v --cgo --prefix ./
```

Comment:
Why not just add the flag to the build target instead of this

Reply:
Probably because the build target easily gets overwritten by upstream merges? I guess this lower the chance of it happening again.

Reply:
Hmm, we have very manual process for maintaining this fork anyway, so probably can keep an eye on that? Also if something in upstream builds do change, better for us to reflect them I think

Reply:
there are these architecture jobs that fail when I add the cgo flag

Reply:
I see. So I'd add a comment here on the dockerfile stating why we use promu instead of make common-build. 
Aside from that LGTM